src/SFA.DAS.AANHub.Api.UnitTests/Controllers/ProfilesControllerTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Api.Controllers;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
+using SFA.DAS.AANHub.Application.Profiles.Queries.GetProfiles;
+using SFA.DAS.AANHub.Application.UnitTests;
+
+namespace SFA.DAS.AANHub.Api.UnitTests.Controllers
+{
+    public class ProfilesControllerTests
+    {
+        [Test]
+        [AutoMoqData]
+        public async Task GetProfiles_InvokesQueryHandler_ReturnsProfiles(
+            [Frozen] Mock<IMediator> mediatorMock,
+            [Greedy] ProfilesController sut)
+        {
+            string userType = "Apprentice";
+            var response = new ValidatedResponse<List<ProfileModel>>
+            (
+                new List<ProfileModel>()
+            );
+
+            mediatorMock.Setup(m => m.Send(It.Is<GetProfilesByUserType>(q => q.UserType == userType), It.IsAny<CancellationToken>()))
+                .ReturnsAsync(response);
+
+            var getResult = await sut.GetProfilesByUserType(userType);
+
+            var result = getResult as OkObjectResult;
+            result.Should().NotBeNull();
+            Assert.AreEqual(StatusCodes.Status200OK, result!.StatusCode);
+        }
```

Comment:
We could add a few more tests for NotFound. There are a few examples in the ControllerTests for the other entities - i.e. Apprentice within ApprenticesControllerTests.cs

Reply:
Implemented